airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/oauth.py

```diff
@@ -92,14 +92,34 @@ def get_grant_type(self) -> InterpolatedString:
     def get_refresh_request_body(self) -> Mapping[str, Any]:
         return self._refresh_request_body.eval(self.config)
 
+    def refresh_access_token(self) -> Tuple[str, Any]:
+        """
+        Returns the refresh token and its lifespan in seconds
```

Comment:
this comment doesn't match the actual interface

Reply:
thanks for the callout, copy pasta